content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,54 +1,46 @@
 ======================================
-Allow Customers to Close their Tickets
+Allow customers to close their tickets
 ======================================
 
 Allowing customers to close their tickets gives them autonomy and minimize misunderstandings about
-when an issue is considered solved, or not. It makes communication and actions more efficient.
+when an issue is considered solved or not.
 
 Configure the feature
 =====================
 
-To configure the feature go to :menuselection:`Helpdesk --> Settings --> Helpdesk Teams --> Edit`
-and enable *Ticket closing*.
+To configure the feature, go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`,
+click on the team, and click :guilabel:`Edit` and enable :guilabel:`Ticket closing`.
```

Comment:
Reads better with time order language and a little more specificity
Wording/flow suggestions:
```suggestion
Get started by navigating to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and
choose the appropriate team. Then click :guilabel:`Edit` and enable :guilabel:`Ticket closing` by
checking the field box.
```